test/extensions/quic_listeners/quiche/platform/spdy_platform_test.cc

```diff
@@ -26,6 +30,14 @@ TEST(SpdyPlatformTest, SpdyArraysize) {
   EXPECT_EQ(5, SPDY_ARRAYSIZE(array));
 }
 
```

Comment:
Can we move these tests under namespace spdy? ditto to Http2PlatformTest

Reply:
Move their namespaces SGTM, but it is unrelated to this PR, let's do it separately.